packages/core/src/account/base.ts

```diff
@@ -51,8 +52,9 @@ export abstract class BaseSmartContractAccount<
     | PublicErc4337Client<TTransport>
     | PublicErc4337Client<HttpTransport>;
 
-  constructor(params: BaseSmartAccountParams<TTransport>) {
-    createBaseSmartAccountParamsSchema<TTransport>().parse(params);
+  constructor(params_: BaseSmartAccountParams<TTransport>) {
```

Comment:
nit[learn]: why underscore after?

Reply:
oh because I wanted to use the `parsed` params instead

Reply:
right I guess I meant as opposed to calling the constructor arg `_params`

Reply:
ohhhh, `_params` implies not used